### PR TITLE
DCS-1341 upgrade to Node 16

### PR DESCRIPTION
**Pull request by GurnankCheema**
Updated Node because of issues highlighted in security report

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:14.17-buster as builder
+FROM node:16.13-bullseye-slim as builder
```
Comment (andrewrlee) [inline Dockerfile:1]: another reference lower down

Comment (GurnankCheema) [reply Dockerfile:1]: done

File: Dockerfile [hunk 2]
```diff
@@ -24,7 +24,7 @@ RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit && \
 
 RUN npm prune --production
 
-FROM node:14.17-buster-slim
+FROM node:16.13-buster-slim
```
Comment (andrewrlee) [inline Dockerfile:27]: bullseye

Comment (GurnankCheema) [reply Dockerfile:27]: indeed!

File: .circleci/config.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@ jobs:
   build:
     executor:
       name: hmpps/node
-      tag: 14.15-browsers
+      tag: 16.13-browsers
```
[the remark this answers is not in the document]
Comment (GurnankCheema) [reply .circleci/config.yml]: done

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**